packages/babel-plugin-transform-runtime/package.json

```diff
@@ -35,6 +35,7 @@
     "@babel/plugin-transform-typeof-symbol": "^7.10.4",
     "@babel/preset-env": "^7.10.4",
     "@babel/runtime": "^7.10.5",
+    "@babel/runtime-corejs3": "^7.10.5",
```

Comment:
This is meant to fix the e2e error: https://app.circleci.com/pipelines/github/babel/babel/3524/workflows/90153e91-f615-4930-8a0f-5ed243d3a6cc/jobs/26605/parallel-runs/0/steps/0-103

The error is thrown because `@babel/runtime-corejs3` is _not_ hoisted to the top level in `e2e-test`. The other test is passing because `@babel/runtime-corejs3` is hoisted to the top level from fixture locations, this is not guaranteed and dev deps should be declared.

Reply:
Thanks!